src/services/FilteringByStatus.tsx

```diff
@@ -0,0 +1,15 @@
+import { Status } from '../types/Status';
+import { Todos } from '../types/Todos';
+
+export const filterTodoByStatus = (todoItems: Todos[], values: Status) => {
+  const copy = [...todoItems];
+
+  switch (values) {
+    case Status.Active:
+      return copy.filter((todo: Todos) => todo.completed !== true);
```

Comment:
1. there is no need to create a copy because `filter` method does not mutate the original arr
2. also, you can write shorter

for `active`:
```
return todoItems.filter(todo => !todo.completed);
```

for `completed`:
```
 return todoItems.filter(todo => todo.completed);
```